JAR_VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
ok! I wonder how we manage jar versions in other plugins?! this seems like a reasonable strategy. the other strategy would be to no have any versions number on the jar since it is not published outside the plugin?